charts/timed/templates/ingress.yaml

```
@@ -17,25 +17,35 @@ spec:
         paths:
           - path: /admin/
             backend:
-              serviceName: {{ $.Values.backend.service.name }}
-              servicePort: {{ $.Values.backend.service.externalPort }}
+              service:
+                name: {{ $.Values.backend.service.name }}
+                port:
+                  number: {{ $.Values.backend.service.externalPort }}
```

I'd recommend to change to named ports instead of using the port number:
```suggestion
                  name: {{ $.Values.backend.service.name }}
```

---

changed to named ports

---

`$.Values.backend.service.name` contains the name of the service and not the port. Meethinks this should be hardcoded `http` instead.

it's hardcoded in the deployment as well:

https://github.com/adfinis-sygroup/helm-charts/blob/448b4536de7f8eb54ada4abc8bcb922fdec29cbb/charts/timed/templates/deployment-backend.yaml#L47